nvme-print-json.c

```diff
@@ -4787,3 +4786,34 @@ struct print_ops *nvme_get_json_print_ops(nvme_print_flags_t flags)
 	json_print_ops.flags = flags;
 	return &json_print_ops;
 }
+
+void obj_add_byte_array(struct json_object *o, const char *k, unsigned char *buf, int len)
+{
+	int i;
+
+	_cleanup_free_ char *value = NULL;
+
+	if (!o || !k || !buf || !len)
+		return;
+
+	value = calloc(1, (len + 1) * 2 + 1);
+
+	if (!value) {
+		obj_add_str(o, k, "Could not allocate string");
+		return;
+	}
+
+	sprintf(value, "0x");
```

Comment:
Isn't this being overwritten by the hex characters of the first byte?

Reply:
Right. Sorry the fix was not correct. Thanks for your suggestion.

Reply:
Fixed the overwritten error.